addons/overheating/stringtable.xml

```diff
@@ -890,6 +891,7 @@
             <Korean>오픈 볼트</Korean>
             <Russian>Открыть болт</Russian>
             <French>Obturateur ouvert</French>
+            <German>Öffne den Verschluss</German>
```

Comment:
Definitely wrong, it's not an action, as in "opening the bolt".
We are talking about the bolt type here, so if the weapons fires from an open bolt (many SMGs and MGs do) vs closed bolt (rifles usually).